documentation/01-ai-core-azure-openai-proxy/05-test-deployed-service.md

```diff
@@ -6,10 +6,12 @@ Now that the inference service (proxy) is running, send a request with the promp
 > When it comes to an inference request, make sure to change the **engine** to your deployment id of the respected OpenAI service from Azure.
 
 ```python
+
 legacy_davinci = False # set True if you have a davinci model deployment on Azure OpenAI Services
 if legacy_davinci:
     body = {
-        "engine": "<YOUR ENGINE>", # include your davinci engine from a deployment of an Azure OpenAI services model
+        "engine": "<YOUR ENGINE>", # The deployment name you chose when you deployed the ChatGPT or GPT-4 model.
```

Comment:
this is for davinci 003 only and the second one is for "ChatGPT" aka gpt-35-turbo or GPT-4